webpack.config.js

```diff
@@ -53,8 +53,7 @@ const FrontendConfig = {
 };
 
 /**
- * This is a temporary config for building the payment methods integration script until it can be
- * moved into the payment extension(s).
+ * Config for building the payment methods integration scripts.
```

Comment:
The "temporary" comment still applies here I think - so maybe worth adding it back. E.g. something like:

`Eventually payment method code will move to relevant extensions.`

cc @nerrad might have good ideas for how to word this.

Reply:
Hmm, maybe not temporary – the core payment methods (COD, cheque etc) might stay here forever?

Reply:
Ah, yes. I wanted to comment this line but later forgot about it. :facepalm: 

> Hmm, maybe not temporary – the core payment methods (COD, cheque etc) might stay here forever?

Yes, that was my thinking too. Stripe integration might be temporary, but support for WC Core methods will stay, correct?